Case-2/Review/questao_3_review.py

```diff
@@ -0,0 +1,19 @@
+def questao_3(lista):
+
+    soma = 0
+    for i in lista:
+        if i != 2*lista.index(i):
+            soma += i
+        else:
+            return soma
```

Comment:
Assim como na questao 1 nao tem necessidade desse else, dexaria de forma semelhante a anterior
```suggestion
        if i == 2*lista.index(i): return soma
        soma += i
```